xrpl/utils/txn_parser/utils/transaction.py

```diff
@@ -0,0 +1,196 @@
+"""Helper functions to format raw transaction data."""
+
+from typing import Any, Dict, List, Union, cast
+
+from pydash import map_  # type: ignore
+from typing_extensions import Literal
+
+from xrpl.utils.txn_parser.utils.types import (
+    AccountBalance,
+    NormalizedFields,
+    NormalizedNode,
+    RawTxnType,
+    SubscriptionRawTxnType,
+    XRPLTxnFieldsException,
+)
+
+
+def validate_transaction_fields(
+    transaction_data: Union[RawTxnType, SubscriptionRawTxnType],
+) -> None:
+    """
+    Check if the transaction fields are valid.
+
+    Args:
+        transaction_data: The raw transaction data.
+
+    Raises:
+        XRPLTxnFieldsException: If the raw transaction data is malformed.
+    """
+    if "transaction" in transaction_data:
+        if "Account" not in transaction_data["transaction"]:  # type: ignore
+            raise XRPLTxnFieldsException(
+                "Malformed transaction fields: Transaction"
+                "field 'Account' must be included."
+            )
+    elif "Account" not in transaction_data:
+        raise XRPLTxnFieldsException(
+            "Malformed transaction fields: Transaction"
+            "field 'Account' must be included."
+        )
+    if "meta" not in transaction_data:
+        raise XRPLTxnFieldsException(
+            "Malformed transaction fields: Transaction field 'meta' must be included."
+        )
+    meta = transaction_data["meta"]
+    if "AffectedNodes" not in meta or not meta["AffectedNodes"]:
+        raise XRPLTxnFieldsException("Malformed transaction fields: No nodes provided.")
+
+
+def normalize_transaction(
+    transaction_data: SubscriptionRawTxnType,
+) -> RawTxnType:
+    """
+    Formats the raw transaction data into one standard format.
+
+    Args:
+        transaction_data: The raw transaction data.
+
+    Returns:
+        Transaction data in standard format.
+    """
+    transaction = transaction_data["transaction"]
+    normalized_txn = {
+        tx_field_name: tx_field_value
+        for tx_field_name, tx_field_value in transaction.items()
+    }
+    meta = transaction_data["meta"]
+    normalized_txn["meta"] = meta
```

Comment:
nit: This should just be one line
```suggestion
    normalized_txn["meta"] = transaction_data["meta"]
```